### ISSUE TITLE
[a11y-defect-2]: Benefit Letter download buttons must have unique text

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#96339]
### Point of contact

Trevor Pierce

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

All the Benefit Letters download buttons say "Download letter" This doesn't give context to what letter and could be problematic for screen reader users. If a screen reader user has all accordions open and is viewing their Form Controls menu, they'll hear "Download letter, button" repeated multiple times. This can cause a significant cognitive load. Screenshot attached.

---
![Image](https://github.com/user-attachments/assets/63442073-b967-44cf-8370-c3fdbc552639)

### Reproduction steps

1. Log into https://staging.va.gov/records/download-va-letters/letters/letter-list with a test user. I used 54.
2. Turn on the screen reader of your choosing
3. Expand an accordion and set focus on the download button
4. Verify the screen reader announces "Download letter, button"
5. Repeat steps 3 and 4 on another accordion

### Proposed solution or next steps

I'd like to suggest making each button name unique and reference the letter in the accordion heading. This will need some design and/or content help as some of the names are very long.

Solution:  Update the button to show the type of the letter being downloaded, followed by the format of the file in parenthesis.  Example: [Proof of Service Letter (PDF)](https://github.com/orgs/department-of-veterans-affairs/projects/1430/views/9?visibleFields=%5B%22Title%22%2C%22Assignees%22%2C%22Status%22%2C138592190%2C%22Labels%22%5D&pane=issue&itemId=85695108&issue=department-of-veterans-affairs%7Cva.gov-team%7C96339#)

### References, articles, or WCAG support

1. Understanding [SC 2.4.4](https://w3.org/TR/WCAG22#link-purpose-in-context): Link Purpose (In Context) (Level A)
2. Understanding [SC 1.3.1](https://w3.org/TR/WCAG22#info-and-relationships): Info and Relationships (Level A)

### Type of issue

- [ ] Axe-core
- [x] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [x] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [x] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else

**Comment by dysbo** [comment]
We would like content/design guidance on what we should be putting into these buttons.

**Comment by rallencf** [comment]
I suggest that each "Download letter" button uses the text of the document title and specifies the file type. For example:

[Commissary Letter (PDF)](#)
[Proof of Service Letter (PDF)](#)

@dysbo My IA/content review of the benefit letter pages will conclude with additional suggested edits on this page. I mention this in case you want to wait to implement this update to do it with the other updates.

**Comment by rallencf** [comment]
@dysbo Yes, suggest removing "download."
